src/main/java/com/jfrog/ide/common/tree/Issue.java

```diff
@@ -0,0 +1,141 @@
+package com.jfrog.ide.common.tree;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author yahavi
+ */
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class Issue extends VulnerabilityOrViolation {
+
+    private Severity severity = Severity.Normal;
+    private List<String> fixedVersions;
+    private List<String> infectedVersions;
+    private List<String> references;
+    private Cve cve;
+    private String summary;
+    private String issueId;
+    private String lastUpdated;
+    private List<String> watchNames;
+    private ResearchInfo researchInfo;
+
+    public Issue() {
+    }
+
+    @SuppressWarnings("unused")
+    public Issue(String issueId, Severity severity, String summary, List<String> fixedVersions, List<String> infectedVersions,
+                 Cve cve, String lastUpdated, List<String> watchNames, List<String> references, ResearchInfo researchInfo) {
+        this.issueId = issueId;
+        this.severity = severity;
+        this.summary = summary;
+        this.fixedVersions = fixedVersions;
+        this.infectedVersions = infectedVersions;
+        this.cve = cve;
+        this.lastUpdated = lastUpdated;
+        this.watchNames = watchNames;
+        this.references = references;
+        this.researchInfo = researchInfo;
+    }
+
+    public String getIssueId() {
+        return this.issueId;
+    }
+
+    public Severity getSeverity() {
+        return this.severity;
+    }
+
+    @SuppressWarnings("unused")
+    public String getSummary() {
+        return summary;
+    }
+
+    @SuppressWarnings("unused")
+    public List<String> getFixedVersions() {
+        return fixedVersions;
+    }
+
+    @SuppressWarnings("unused")
+    public void setFixedVersions(List<String> fixedVersions) {
+        this.fixedVersions = fixedVersions;
+    }
+
+    public List<String> getInfectedVersions() {
```

Comment:
There are more unused in this class
```suggestion
    @SuppressWarnings("unused")
    public List<String> getInfectedVersions() {
```